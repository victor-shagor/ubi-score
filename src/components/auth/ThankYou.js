import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import Box from "@material-ui/core/Box";
import Button from "components/Button";
import logo from "assets/images/logo.png";

const useStyles = makeStyles({
  wrapper: {
    backgroundColor: "#f6f7f8",
    height: "100vh",
    width: "50vw",
    padding: "0 43px 43px 43px",
  },
  header: {
    fontFamily: "EpilogueBold",
    fontSize: "28px",
    marginTop: "70px",
    marginBottom: "8px",
  },
  detail: { fontFamily: "outfit", opacity: 0.6, marginBottom: "50px" },
  read: {
    marginRight: "10px",
    color: "#6D707A",
  },
  link: { color: "#1F8EDE", fontFamily: "EpilogueSemiBold" },
});

const ThankYou = () => {
  const classes = useStyles();

  return (
    <div className={classes.wrapper}>
      <Box
        height="90%"
        paddingLeft="45px"
        paddingTop="58px"
        paddingRight="77px"
        bgcolor="white"
        borderRadius="0px 0px 20px 20px"
      >
        <img src={logo} alt="logo" />
        <p className={classes.header}>Thank You</p>
        <p className={classes.detail}>
          Amet minim mollit non deserunt ullamco est sit aliqua dolor{" "}
        </p>
        <Button text="Back to Home" width="256px" bgColor="#4285DE" />
        <p className={classes.read}>
          Question? Email us
          <a className={classes.link} href="email">
            {" "}
            Login
          </a>
        </p>
      </Box>
    </div>
  );
};

export default ThankYou;
