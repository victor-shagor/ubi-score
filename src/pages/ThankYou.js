import React from "react";
import LeftPanel from "components/auth/leftPanel";
import RightPanel from "components/auth/ThankYou";

const ThankYou = () => {
  return (
    <div style={{ display: "flex" }}>
      <LeftPanel
        name="Jerome Bell"
        role="Product Designer"
        text="“Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit”"
      />
      <RightPanel />
    </div>
  );
};

export default ThankYou;
