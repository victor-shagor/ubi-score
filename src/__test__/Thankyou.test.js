import { mount, shallow } from "enzyme";
import toJson from "enzyme-to-json";
import ThankYou from "components/auth/ThankYou";

it("renders without crashing", () => {
  shallow(<ThankYou />);
});

it("should render two paragraphs", () => {
  const wrapper = mount(<ThankYou />);
  expect(wrapper.find("p").length).toEqual(3);
});

it("should render two paragraphs", () => {
  const wrapper = mount(<ThankYou />);
  expect(wrapper.find("a").length).toEqual(1);
});

it("renders correctly", () => {
  const tree = shallow(<ThankYou />);
  expect(toJson(tree)).toMatchSnapshot();
});
