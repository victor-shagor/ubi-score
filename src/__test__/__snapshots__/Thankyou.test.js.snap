// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="makeStyles-wrapper-1"
>
  <Styled(MuiBox)
    bgcolor="white"
    borderRadius="0px 0px 20px 20px"
    height="90%"
    paddingLeft="45px"
    paddingRight="77px"
    paddingTop="58px"
  >
    <img
      alt="logo"
      src="logo.png"
    />
    <p
      className="makeStyles-header-2"
    >
      Thank You
    </p>
    <p
      className="makeStyles-detail-3"
    >
      Amet minim mollit non deserunt ullamco est sit aliqua dolor
       
    </p>
    <Button
      bgColor="#4285DE"
      text="Back to Home"
      width="256px"
    />
    <p
      className="makeStyles-read-4"
    >
      Question? Email us
      <a
        className="makeStyles-link-5"
        href=""
      >
         
        Login
      </a>
    </p>
  </Styled(MuiBox)>
</div>
`;
